src/patient-registration/widgets/section/person-attribute-section.component.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { render, fireEvent, wait, screen, cleanup } from '@testing-library/react';
+import { useConfig } from '@openmrs/esm-module-config';
+import { ConfigMock } from '../../../../__mocks__/openmrs-esm-config.mock';
+import { mockCurrentUserLocation } from '../../../../__mocks__/currentUserLocation.mock';
+import { mockUniquePatientIdentifiers } from '../../../../__mocks__/uniquePatientIdentifiers.mock';
```

Comment:
Here, too, having the mock in a separate file just makes the code harder to understand. The mock is very short. Just include it directly in this file.